### ISSUE TITLE
Change routes to separate files

### ISSUE THREAD

**Issue by chanakaDe** [chanakaDe/Mint-REST#5]
Now all the routes are in a same file. 
https://github.com/chanakaDe/Mint-REST/blob/master/app/routes/api.js 
All in api.js file. User and Story controllers must be in a different files.
Have to fix that. :+1:

**Comment by saravanan10393** [comment]
working on it.

**Comment by chanakaDe** [comment]
@saravanan10393  How's the work ? Done ? Working well ?

**Comment by saravanan10393** [comment]
Yes it is. You could check it by following. Updated read me file.

**Comment by saravanan10393** [comment]
HI, can you close this issue since it is already completed.
